qa/tasks/cephfs/fuse_mount.py

```diff
@@ -424,20 +425,24 @@ def find_socket(client_name):
         for f in files:
                 pid = re.match(".*\.(\d+)\.asok$", f).group(1)
                 if os.path.exists("/proc/{{0}}".format(pid)):
-                        return f
+                    with open("/proc/{{0}}/cmdline".format(pid), 'r') as proc_f:
+                        contents = proc_f.read()
+                        if mountpoint in contents and 'sudo' not in contents:
```

Comment:
And `'sudo' not in contents` is no need anymore. Since we find the root cause, which is the `guest` in the asok path.